XROLE/storeManus-React
# [Chore: #163671366]Setup React router v4

#### What does this PR do?
- Setup React Router version 4

#### Description of task completed
- install react-router and react-router-dom
- add routing functionality
- create two more component to test routing

#### How should thisl be tested?
- run npm install and npm start
- navigate to GET `/login` to see the login page

#### What are pivotal story associated with this?
[#163671334](https://www.pivotaltracker.com/story/show/163671334)
